app/controllers/adventure_logs_controller.rb

```diff
@@ -17,12 +17,25 @@ def create
     end
   end
 
+  def update
+    previous = GameSession.find(game_session_id).best_adventure_log
+    previous.update(best: false) if previous && previous.id != adventure_log_id
```

Comment:
This is a really good approach. When I was thinking about this in my head, I was planning on just setting all adventure logs for that session and then setting them to not best and then setting the one I wanted as the best.

Reply:
Thanks! I started out that way.